mcp/ARCHITECTURE.md

```
@@ -0,0 +1,55 @@
+# CNG MCP Server Architecture
+
+## Overview
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)